### PR TITLE
Enable group calling V3

### PR DESCRIPTION
**Pull request by basine**

### CODE REVIEW

**Comment by vytis** [comment]
Tests failed for some reason...

File: Wire-iOS/Sources/UserInterface/Overlay/VoiceChannelController.m [hunk 1]
```diff
@@ -205,7 +210,7 @@ - (void)updateActiveCallConversation
 - (void)callCenterDidChangeVoiceChannelState:(VoiceChannelV2State)voiceChannelState conversation:(ZMConversation *)conversation callingProtocol:(enum CallingProtocol)callingProtocol
 {
     DDLogVoice(@"SE: Voice channel state change to %d (%@)", voiceChannelState, StringFromVoiceChannelV2State(voiceChannelState));
-    [self updateOverlays];
+    [self updateOverlaysWithOutgoingCallInConversation: voiceChannelState == VoiceChannelV2StateOutgoingCall ? conversation : nil];
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Overlay/VoiceChannelController.m]: Is this some kind work-around?

**Comment by mikeger** [comment]
@basine as always, some mock in tests expired 💥

**Comment by mikeger** [comment]
I guess we better switch the old tests to use the real CoreData

File: Wire-iOS/Sources/UserInterface/Overlay/VoiceChannelController.m (hunk 1 above)
Comment (basine) [inline Wire-iOS/Sources/UserInterface/Overlay/VoiceChannelController.m]: Yeah, for outgoing calls, AVS does not return the outgoing call (yet) when we notify the UI - This is probably due to threading issues. Looking for other solutions for this, too. :-/

**Pull request merged**